docs/howto/add_relationship_links.md

```diff
@@ -104,12 +104,7 @@ class Api::V1::UserSerializer < ActiveModel::Serializer
 
   has_many :microposts, serializer: Api::V1::MicropostSerializer do
     link(:related) { api_v1_microposts_path(user_id: object.id) }
-  end
-
-  #this is needed to avoid n+1, gem core devs are working to remove this necessity
-  #more on: https://github.com/rails-api/active_model_serializers/issues/1325
-  def microposts
-    object.microposts.loaded ? object.microposts : object.microposts.none
+    include_data false
```

Comment:
@richmolj I would say the opposite is more often: if you add links you probably don't include the relationship.

I would suggest to merge this request and you can start a new one about conditionally adding data (probably in another section of the wiki)